### DISCUSSION TITLE
All the flags of YouTube in each version (start from v17.39.5)

### DISCUSSION THREAD

**Discussion by qnblackcat** [PoomSmart/YTABConfig#2] [General]
### To determine the new flags that have been added in each release, simply copy all the flags of the two versions, and then throw them into a diff checker.
- YouTube **v17.39.5** (1661 feature flags): https://gist.github.com/qnblackcat/5558e1ec0e5ebe84692632311b973aab
- YouTube **v17.40.5** (1687 feature flags): https://gist.github.com/qnblackcat/e063738c82de5db11ac4b56f76934a9d
- YouTube **v17.41.2** (1711 feature flags): https://gist.github.com/qnblackcat/6a386ab77b7e99fdaf97177d2320a75c
- YouTube **v17.42.6** (1725 feature flags): https://gist.github.com/qnblackcat/25ca0fb0e67be403de40c928562f6329

**Comment by js1darren** [comment]
> ### To determine the new flags that have been added in each release, simply copy all the flags of the two versions, and then throw them into a diff checker.
> 
> - YouTube **v17.39.5** (1661 feature flags): https://gist.github.com/qnblackcat/5558e1ec0e5ebe84692632311b973aab
> 
> - YouTube **v17.40.5** (1687 feature flags): https://gist.github.com/qnblackcat/e063738c82de5db11ac4b56f76934a9d
> 
> - YouTube **v17.41.2** (1711 feature flags): https://gist.github.com/qnblackcat/6a386ab77b7e99fdaf97177d2320a75c
> 
> - YouTube **v17.42.6** (1725 feature flags): https://gist.github.com/qnblackcat/25ca0fb0e67be403de40c928562f6329
